nbconvert/exporters/html.py

```diff
@@ -150,6 +152,28 @@ def resources_include_css(name):
             code = """<style type="text/css">\n%s</style>""" % (env.loader.get_source(env, name)[0])
             return jinja2.Markup(code)
 
+        def include_lab_theme(name):
+            # Try to find the theme with the given name, looking through the labextensions
+            paths = jupyter_path('labextensions')
+
+            for path in paths:
+                for (dirpath, dirnames, filenames) in os.walk(path):
+                    # If it's a federated labextension that contains themes
+                    if 'package.json' in filenames and 'themes' in dirnames:
```

Comment:
Not sure yet whether this would be better, but an alternative could be to look for the `themePath` field in the `package.json`: 

https://github.com/jupyterlab/jupyterlab/blob/6911da2af507a73add0ae79469d806ea85118662/builder/metadata_schema.json#L76-L81

Reply:
Indeed, I wasn't sure if I needed to use that. Isn't the `themePath` a relative path in the development repository structure? Does it still make sense in the generated federated extension?

Reply:
Right, it's probably better to stick to the approach this current PR proposes.

Which should be similar to what `jupyterlab_server` does: https://github.com/jupyterlab/jupyterlab_server/blob/main/jupyterlab_server/themes_handler.py

Reply:
Thanks for the link! I assumed `index.css` would always be the entry point in the PR, is that a good assumption?

Reply:
Probably this could be a different name? Although in the theme cookiecutter it's true it defaults to `index.css`: https://github.com/jupyterlab/theme-cookiecutter/blob/e95554b339bec0b4d44ee265a9d861f27e7db276/%7B%7Bcookiecutter.python_name%7D%7D/src/index.ts#L17